tests/test/adremover-loggedin-erc20.test.js

```diff
@@ -0,0 +1,193 @@
+const url = require('../helpers/url')
+const wait = require('../helpers/wait')
+// const debug = require('../helpers/debugging')
+const iframes = require('../helpers/iframes')
+const { adblockERC20LockAddresses } = require('../helpers/vars')
+
+// const it = debug.screenshotOnFail(page)
+
+let lockSelectors
+/*
+to test locally, paste this into the console:
```

Comment:
Do you mind explaining why?

Reply:
I'll clarify the comment. If the integration test fails, and you want to know why, the easiest way to debug is not to run the integration tests locally, but to load the same URL the integration test uses.

Since the locks are deployed in the dev standup, this becomes possible!

However, constructing that exact URL is pretty complex if you don't know how. The stuff in that comment is how to easily construct the URL needed. So, you can open safari, paste that into the console and it will load the file.

I used that process to debug the test, so I figured I might as well document it.